docs/install-nuget-client-tools.md

```diff
@@ -38,6 +38,25 @@ The [MSBuild CLI](reference/msbuild-targets.md) also restores and creates packag
 
 Package Manager Console commands work only within Visual Studio on Windows and don't work within other PowerShell environments.
 
+## Support policy
```

Comment:
I think the support policy should cover other NuGet distribution vehicles such as VS, .NET SDK, NuGet Client SDK (pointing to it's the dedicated support policy link) also in addition to `nuget.exe`. Please refer to https://devblogs.microsoft.com/nuget/announcing-nuget-exe-and-nuget-client-sdk-packages-support-policy-keeping-you-informed-and-secure/#what-you-can-expect

Reply:
Could you specify what you want said? According to Andy, the blog is wrong about the Modern Lifecycle Policy applying to the SDK. I already removed that.

Reply:
Doesn't this sentence explain that for nuget.exe?

> For more information on NuGet.exe's support policy, see the [Microsoft Modern Lifecycle Policy](https://aka.ms/lifecycle).

Reply:
My suggestion is to cover all the distribution vehicles here not only NuGet.exe. NuGet Client tools are distributed through the VS, .NET SDK, NuGet.exe and NuGet SDK. Instead of covering NuGet.exe we should also add a link to support policy of VS, .NET SDK, NuGet Client SDK along with NuGet.exe. Please refer to the table in the blog post https://devblogs.microsoft.com/nuget/announcing-nuget-exe-and-nuget-client-sdk-packages-support-policy-keeping-you-informed-and-secure/#what-you-can-expect

Reply:
Please see what you think of my current change.